src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -205,34 +155,32 @@ const FIELDS = {
         type: params => (params.subfield ? <LabelField {...params} /> : null),
         label: 'react.partialReceiving.code.label',
         defaultMessage: 'Code',
-        headerAlign: 'left',
-        flexWidth: '0.8',
-        attributes: {
-          className: 'text-left ml-1',
-        },
       },
       'product.name': {
         type: params => (params.subfield ? <LabelField {...params} /> : null),
         label: 'react.partialReceiving.product.label',
         defaultMessage: 'Product',
         headerAlign: 'left',
-        flexWidth: '3.3',
         attributes: {
-          className: 'text-left ml-1',
-          showValueTooltip: true,
+          className: 'text-left',
+          formatValue: value => (
+            <span className="d-flex">
+              <span className="text-truncate">
+                {value}
+              </span>
+            </span>
+          ),
         },
       },
       lotNumber: {
         type: params => (params.subfield ? <LabelField {...params} /> : null),
         label: 'react.partialReceiving.lotSerialNo.label',
-        defaultMessage: 'Lot/Serial No.',
-        flexWidth: '1',
+        defaultMessage: 'Lot',
```

Comment:
You should also modify the react.partialReceiving.lotSerialNo.label in message.properties